robottelo/host_helpers/satellite_mixins.py

```diff
@@ -107,6 +110,42 @@ def md5_by_url(self, url):
             f'wget -qO - {url} | tee {filename} | md5sum | awk \'{{print $1}}\''
         ).stdout
 
+    def upload_manifest(self, org_id, manifest=None, interface='API', timeout=None):
+        """Upload a manifest with locking, using the requested interface.
+
+        :type org_id: int
+        :type manifest: robottelo.manifests.Manifest
```

Comment:
For now this is just not `robottelo.manifests.Manifest`, its the manifester object as well.

Reply:
Updated.